icu4c/source/i18n/units_converter.h

```diff
@@ -94,20 +94,18 @@ struct U_I18N_API ConversionInfo {
 void U_I18N_API addSingleFactorConstant(StringPiece baseStr, int32_t power, Signum sigNum,
                                         Factor &factor, UErrorCode &status);
 
+// TODO: Make all the fields of `ConversionRate` constants.
```

Comment:
mmm, I prefer to make it immutable class, but we can discuss this while we are solving the TODO in another PR

Reply:
If you make the change I suggested above, hoisting field initialization out of the constructor's function block, you might be able to put the `const` back.

Reply:
yes, but still I prefer to make all the const. I will consider making all of them const or just make the variable const in another PR.

Done

Reply:
It's not necessary to make scalar fields const, because `const ConversionRate` implies that all the scalar fields are also const.  You need the extra `const` on the `MeasureUnitImpl` because it is an object.

Reply:
okay, done